CONTRIBUTING.md

```diff
@@ -9,3 +9,17 @@ This project uses a Lerna Monorepo. Lerna is a tool used to manage multiple Java
 1. At the root level of the project, run `yarn`. This will install the packages required for Lerna to work, as well as automatically install packages for the frontend and backend packages.
 
 2. Once again at the project root, run `npm run start`. This will automatically run the `start` command in all nested packages, which will simultaneously run the frontend and backend.
+
+# Firebase Setup
+
+This project uses Firebase for authentication. You may want to set up your own Firebase project for development of this application.
```

Comment:
Can we setup a Firebase project for the whole group?
Is it going to cause conflict merging back to main repo if everyone has their own config in FirebaseConfigPrinter.js

Reply:
It is currently setup for our group (the current one is in env.development, which is a real project). The reason for this stuff is the next group.

Firebase admin will need to be setup on the backend which requires a private key, which they won't be able to access since they won't have any permissions.

I did not consider the issue of conflicts, I'll modify this script a bit to avoid that.